bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml

```diff
@@ -183,6 +188,17 @@
 		<label>Led Status</label>
 		<state pattern="%.0f" readOnly="true"/>
 	</channel-type>
+	<channel-type id="mop_drying_time">
+		<item-type>Number</item-type>
+		<label>Mop Drying Time</label>
+		<state pattern="%.0f %unit%" readOnly="true"/>
```

Comment:
The channel won't have a unit unless it's defined as UoM. Since you are already updating it correctly with QuantityType with unit, you could use `Number:Time` instead.